demos/tabs.php

```diff
@@ -18,6 +18,13 @@
     $tab->add(['LoremIpsum', 'size' => 2]);
 });
 
+// modal tab
+$t->addTab('Modal popup', function ($tab) {
+    $tab->add(['Button', 'Load Lorem'])->on('click', $tab->add('Modal')->set(function ($p) {
+        $p->add(['LoremIpsum', 'size' => 2]);
+    })->show());
+});
```

Comment:
demo don't work well
* try open modal popup, then close it by clicking X and then open it again - it will open modal dialog, but empty without LoremIpsum.
* also there is quite ugly flickering of modal window while it is opening (re-opening), but I think it's because of window being small while it shows loader spinner.

I found out that in cases when no LoremIpsum shows up, it's because it don't make Ajax call to server at all. So it's probably some issue with how JS executes.

Reply:
I'll try to ask @ibelar for help..